src/lightning_app/cli/lightning_cli.py

```diff
@@ -380,6 +381,27 @@ def run_app(
 
     run.add_command(_run_model)
 
+
+@_main.command("open")
```

Comment:
if this is an early access feature, maybe let's hide it from the `help` text. I think passing `hidden=True` here should work?